### PR TITLE
Retire EL6 (SOFTWARE-4383)

### PR DESCRIPTION
**Pull request by brianhlin**
I think this eliminates all the EL6 references in our docs

### CODE REVIEW

File: docs/data/load-balanced-gridftp.md [hunk 1]
```diff
@@ -334,23 +327,16 @@ Using Your Load Balanced GridFTP System
...
 1. Manage the service with the following commands:
 
     | **To ...** | **On EL 6, run the command...** | **On EL 7, run the command...** | 
     | -------- | ----------------------------- | ----------------------------- |
-    | Start a service | `service SERVICE-NAME start` | `systemctl start SERVICE-NAME` | 
-    | Stop a service | `service SERVICE-NAME stop` | `systemctl stop SERVICE-NAME` |
-    | Enable a service to start during boot | `chkconfig SERVICE-NAME on` | `systemctl enable SERVICE-NAME` | 
-    | Disable a service from starting during boot | `chkconfig SERVICE-NAME off` | `systemctl disable SERVICE-NAME` |
+    | Start the service | `service arptables start` | `systemctl start arptables` | 
+    | Stop atheservice | `service arptables stop` | `systemctl stop arptables` |
```
Comment (matyasselmeci) [inline docs/data/load-balanced-gridftp.md:337]: I think you missed this table?
Also replace `Stop atheservice` with `Stop the service`

File: docs/other/gsissh.md [hunk 2]
```diff
@@ -74,12 +74,12 @@ Using GSI OpenSSH
...
-| Start  service                          | `service gsisshd start`      | `systemctl start gsisshd`   |
-| Stop a  service                         | `service gsisshd stop`       | `systemctl stop gsisshd`    |
-| Enable a service to start on boot       | `chkconfig gsisshd on`       | `systemctl enable gsisshd`  |
-| Disable a service from starting on boot | `chkconfig gsisshd off`      | `systemctl disable gsisshd` |
+| To...                                   | Run the command...            |
+| :-------------------------------------- | :---------------------------- |
+| Start  service                          | `systemctl start gsisshd`     |
+| Stop a  service                         | `systemctl stop gsisshd`      |
+| Enable a service to start on boot       | `systemctl enable gsisshd`    |
+| Disable a service from starting on boot | `systemctl disable gsisshd`   |
```
Comment (matyasselmeci) [inline docs/other/gsissh.md:82]:
```suggestion
| Start the service                         | `systemctl start gsisshd`     |
| Stop the service                          | `systemctl stop gsisshd`      |
| Enable the service to start on boot       | `systemctl enable gsisshd`    |
| Disable the service from starting on boot | `systemctl disable gsisshd`   |
```

File: docs/other/install-gwms-frontend.md [hunk 3]
```diff
@@ -674,37 +671,36 @@ Using GlideinWMS
...
-| To...                                   | On EL6, run the command...                  | On EL7, run the command...                      |
-| :-------------------------------------- | :----------------------------------------   | :--------------------------------------------   |
-| Start a service                         | `service <SERVICE-NAME> start` | `systemctl start <SERVICE-NAME>`   |
-| Stop a  service                         | `service <SERVICE-NAME> stop`  | `systemctl stop <SERVICE-NAME>`    |
-| Enable a service to start on boot       | `chkconfig <SERVICE-NAME> on`  | `systemctl enable <SERVICE-NAME>`  |
-| Disable a service from starting on boot | `chkconfig <SERVICE-NAME> off` | `systemctl disable <SERVICE-NAME>` |
+| To...                                   | Run the command...                 |
+| :-------------------------------------- | :--------------------------------  |
+| Start a service                         | `systemctl start <SERVICE-NAME>`   |
+| Stop a  service                         | `systemctl stop <SERVICE-NAME>`    |
```
Comment (matyasselmeci) [inline docs/other/install-gwms-frontend.md:689]:
```suggestion
| Stop a service                          | `systemctl stop <SERVICE-NAME>`    |
```

File: docs/release/supported_platforms.md [hunk 4]
```diff
@@ -11,13 +11,10 @@ for 64-bit Intel architectures according to the following table:
 
 | Platform                   | OSG 3.5  | OSG 3.4  |
```
Comment (matyasselmeci) [inline docs/release/supported_platforms.md:12]: 3.4's out of support -- should we change this to 3.5 and 3.6?

File: docs/security/lcmaps-voms-authentication.md [hunk 5]
```diff
@@ -273,8 +273,8 @@ and make sure it matches one of the patterns in `/etc/grid-security/voms-mapfile
 Troubleshooting the LCMAPS VOMS Plugin
 --------------------------------------
 
-LCMAPS logs to `journalctl` (EL7) or `/var/log/messages` (EL6) and the verbosity of the logging can be increased by
-modifying the appropriate configuration and restarting the relevant service.
+LCMAPS logs to `journalctl` (EL7) and the verbosity of the logging can be increased by modifying the appropriate
```
Comment (matyasselmeci) [inline docs/security/lcmaps-voms-authentication.md:276]:
```suggestion
LCMAPS logs to `journalctl` and the verbosity of the logging can be increased by modifying the appropriate
```

File: docs/security/lcmaps-voms-authentication.md [hunk 6]
```diff
@@ -362,8 +362,8 @@ The default value for this parameter is 12hrs.
 
 #### Wrong version of GridFTP
 
-If you have the EPEL version of the GridFTP server, you may see error messages in `journalctl` (EL7),
-`var/log/messages` (EL6), or the location specified by `LCMAPS_LOG_FILE`.
+If you have the EPEL version of the GridFTP server, you may see error messages in `journalctl` (EL7)
```
Comment (matyasselmeci) [inline docs/security/lcmaps-voms-authentication.md:365]:
```suggestion
If you have the EPEL version of the GridFTP server, you may see error messages in `journalctl`
```

File: docs/worker-node/install-singularity.md [hunk 7]
```diff
@@ -265,10 +263,9 @@ before installing the required packages:
 singularity includes an option called `underlay` that enables using bind
 mount points that do not exist in the container image.
 By default it is enabled, but only if the similar `overlay` option cannot
-be used, such as on RHEL 6 where kernel support for overlayfs is missing
-or when running in unprivileged mode.  On RHEL 7 and RHEL 8 is recommended to
-completely disable `overlay`, because it is more vulnerable to security
-problems than `underlay`.
+be used, such as when running in unprivileged mode. 
+On RHEL 7 and RHEL 8 is recommended to completely disable `overlay`,
```
Comment (matyasselmeci) [inline docs/worker-node/install-singularity.md:267]:
```suggestion
It is recommended to completely disable `overlay`,
```

File: docs/common/ca.md [hunk 8]
```diff
@@ -63,12 +63,12 @@ See [this page](https://opensciencegrid.org/security/CaDistribution/) for detail
...
-    | To...                                   | On EL6, run the command...                  | On EL7, run the command...                      |
-    | :-------------------------------------- | :----------------------------------------   | :--------------------------------------------   |
-    | Start a service                         | `service <SERVICE-NAME> start` | `systemctl start <SERVICE-NAME>`   |
-    | Stop a  service                         | `service <SERVICE-NAME> stop`  | `systemctl stop <SERVICE-NAME>`    |
-    | Enable a service to start on boot       | `chkconfig <SERVICE-NAME> on`  | `systemctl enable <SERVICE-NAME>`  |
-    | Disable a service from starting on boot | `chkconfig <SERVICE-NAME> off` | `systemctl disable <SERVICE-NAME>` |
+    | To...                                   | Run the command...                            |
+    | :-------------------------------------- | :-------------------------------------------- |
+    | Start a service                         | `systemctl start <SERVICE-NAME>`              |
+    | Stop a  service                         | `systemctl stop <SERVICE-NAME>`               |
```
Comment (matyasselmeci) [inline docs/common/ca.md:69]:
```suggestion
    | Stop a service                          | `systemctl stop <SERVICE-NAME>`               |
```

File: docs/common/ca.md [hunk 9]
```diff
@@ -222,12 +222,12 @@ To modify the times that `fetch-crl-cron` runs, edit `/etc/cron.d/fetch-crl`.
...
-| To...                                   | On EL6, run the command...                | On EL7, run the command...                    |
-| :-------------------------------------- | :---------------------------------------- | :-------------------------------------------- |
-| Start a service                         | `service <SERVICE-NAME> start`            | `systemctl start <SERVICE-NAME>`              |
-| Stop a  service                         | `service <SERVICE-NAME> stop`             | `systemctl stop <SERVICE-NAME>`               |
-| Enable a service to start on boot       | `chkconfig <SERVICE-NAME> on`             | `systemctl enable <SERVICE-NAME>`             |
-| Disable a service from starting on boot | `chkconfig <SERVICE-NAME> off`            | `systemctl disable <SERVICE-NAME>`            |
+| To...                                   | Run the command...                            |
+| :-------------------------------------- | :-------------------------------------------- |
+| Start a service                         | `systemctl start <SERVICE-NAME>`              |
+| Stop a  service                         | `systemctl stop <SERVICE-NAME>`               |
```
Comment (matyasselmeci) [inline docs/common/ca.md:228]:
```suggestion
| Stop a service                          | `systemctl stop <SERVICE-NAME>`               |
```

File: docs/compute-element/install-htcondor-ce.md [hunk 10]
```diff
@@ -371,12 +371,12 @@ The specific services are:
...
-| To...                                   | On EL6, run the command...                  | On EL7, run the command...                      |
-| :-------------------------------------- | :----------------------------------------   | :--------------------------------------------   |
-| Start a service                         | `service <SERVICE-NAME> start` | `systemctl start <SERVICE-NAME>`   |
-| Stop a  service                         | `service <SERVICE-NAME> stop`  | `systemctl stop <SERVICE-NAME>`    |
-| Enable a service to start on boot       | `chkconfig <SERVICE-NAME> on`  | `systemctl enable <SERVICE-NAME>`  |
-| Disable a service from starting on boot | `chkconfig <SERVICE-NAME> off` | `systemctl disable <SERVICE-NAME>` |
+| To...                                   | Run the command...                            |
+| :-------------------------------------- | :-------------------------------------------- |
+| Start a service                         | `systemctl start <SERVICE-NAME>`              |
+| Stop a  service                         | `systemctl stop <SERVICE-NAME>`               |
```
Comment (matyasselmeci) [inline docs/compute-element/install-htcondor-ce.md:377]:
```suggestion
| Stop a service                          | `systemctl stop <SERVICE-NAME>`               |
```

File: docs/data/frontier-squid.md [hunk 11]
```diff
@@ -109,12 +109,12 @@ To configure the OSG Compute Entrypoint (CE) to know about your Frontier Squid s
...
-| Start a service                         | `service frontier-squid start` | `systemctl start frontier-squid`   |
-| Stop a  service                         | `service frontier-squid stop`  | `systemctl stop frontier-squid`    |
-| Enable a service to start on boot       | `chkconfig frontier-squid on`  | `systemctl enable frontier-squid`  |
-| Disable a service from starting on boot | `chkconfig frontier-squid off` | `systemctl disable frontier-squid` |
+| To...                                   | Run the command...                            |
+| :-------------------------------------- | :-------------------------------------------- |
+| Start a service                         | `systemctl start frontier-squid`              |
+| Stop a  service                         | `systemctl stop frontier-squid`               |
+| Enable a service to start on boot       | `systemctl enable frontier-squid`             |
+| Disable a service from starting on boot | `systemctl disable frontier-squid`            |
```
Comment (matyasselmeci) [inline docs/data/frontier-squid.md:117]:
```suggestion
| Start the service                         | `systemctl start frontier-squid`              |
| Stop the service                          | `systemctl stop frontier-squid`               |
| Enable the service to start on boot       | `systemctl enable frontier-squid`             |
| Disable the service from starting on boot | `systemctl disable frontier-squid`            |
```

File: docs/data/gridftp.md [hunk 12]
```diff
@@ -185,12 +185,12 @@ In addition to the GridFTP service itself, there are a number of supporting serv
...
-| To...                                   | On EL6, run the command...                  | On EL7, run the command...                      |
-| :-------------------------------------- | :----------------------------------------   | :--------------------------------------------   |
-| Start a service                         | `service <SERVICE-NAME> start` | `systemctl start <SERVICE-NAME>`   |
-| Stop a  service                         | `service <SERVICE-NAME> stop`  | `systemctl stop <SERVICE-NAME>`    |
-| Enable a service to start on boot       | `chkconfig <SERVICE-NAME> on`  | `systemctl enable <SERVICE-NAME>`  |
-| Disable a service from starting on boot | `chkconfig <SERVICE-NAME> off` | `systemctl disable <SERVICE-NAME>` |
+| To...                                   | Run the command...                            |
+| :-------------------------------------- | :-------------------------------------------- |
+| Start a service                         | `systemctl start <SERVICE-NAME>`              |
+| Stop a  service                         | `systemctl stop <SERVICE-NAME>`               |
```
Comment (matyasselmeci) [inline docs/data/gridftp.md:191]:
```suggestion
| Stop a service                          | `systemctl stop <SERVICE-NAME>`               |
```

File: docs/data/install-hadoop.md [hunk 13]
```diff
@@ -470,12 +470,12 @@ Running Services
...
-| To...                                   | On EL6, run the command...                  | On EL7, run the command...                      |
-| :-------------------------------------- | :----------------------------------------   | :--------------------------------------------   |
-| Start a service                         | `service <SERVICE-NAME> start` | `systemctl start <SERVICE-NAME>`   |
-| Stop a  service                         | `service <SERVICE-NAME> stop`  | `systemctl stop <SERVICE-NAME>`    |
-| Enable a service to start on boot       | `chkconfig <SERVICE-NAME> on`  | `systemctl enable <SERVICE-NAME>`  |
-| Disable a service from starting on boot | `chkconfig <SERVICE-NAME> off` | `systemctl disable <SERVICE-NAME>` |
+| To...                                   | Run the command...                            |
+| :-------------------------------------- | :-------------------------------------------- |
+| Start a service                         | `systemctl start <SERVICE-NAME>`              |
+| Stop a  service                         | `systemctl stop <SERVICE-NAME>`               |
```
Comment (matyasselmeci) [inline docs/data/install-hadoop.md:476]:
```suggestion
| Stop a service                          | `systemctl stop <SERVICE-NAME>`               |
```

File: docs/data/xrootd/install-standalone.md [hunk 14]
```diff
@@ -167,18 +163,18 @@ The specific services are:
 | Software         | Service Name                            | Notes                                                                        |
 |:-----------------|:----------------------------------------|:-----------------------------------------------------------------------------|
 | Fetch CRL        | `fetch-crl-boot` and `fetch-crl-cron`   | See [CA documentation](../../common/ca.md#managing-fetch-crl-services) for more info |
-| XRootD           | EL 7:`xrootd@standalone`, EL 6:`xrootd` |                                                                              |
+| XRootD           | EL 7:`xrootd@standalone` | |
 | XRootD Multiuser | EL 7:`xrootd-privileged@standalone`     | See [XRootD multiuser](#enabling-multi-user-support) for more info           |
```
Comment (matyasselmeci) [inline docs/data/xrootd/install-standalone.md:167]:
```suggestion
| XRootD           | `xrootd@standalone` | |
| XRootD Multiuser | `xrootd-privileged@standalone`     | See [XRootD multiuser](#enabling-multi-user-support) for more info           |
```

File: docs/data/xrootd/install-standalone.md [hunk 15]
```diff
@@ -167,18 +163,18 @@ The specific services are:
...
 Start the services in the order listed and stop them in reverse order.
 As a reminder, here are common service commands (all run as `root`):
 
-| To …                                        | On EL 7, run the command…        | On EL 6, run the command…    |
-|:--------------------------------------------|:---------------------------------|:-----------------------------|
-| Start a service                             | `systemctl start SERVICE-NAME`   | `service SERVICE-NAME start` |
-| Stop a service                              | `systemctl stop SERVICE-NAME`    | `service SERVICE-NAME stop`  |
-| Enable a service to start during boot       | `systemctl enable SERVICE-NAME`  | `chkconfig SERVICE-NAME on`  |
-| Disable a service from starting during boot | `systemctl disable SERVICE-NAME` | `chkconfig SERVICE-NAME off` |
+| To …                                        | On EL 7, run the command…        |
```
Comment (matyasselmeci) [inline docs/data/xrootd/install-standalone.md:172]:
```suggestion
| To …                                        | Run the command…                 |
```

**Review by matyasselmeci — APPROVED** [review]
There are a couple of issues but overall fine; pre-approving.

File: docs/data/load-balanced-gridftp.md (hunk 1 above)
Comment (brianhlin) [reply docs/data/load-balanced-gridftp.md]: Good catch. Fixed

File: docs/release/supported_platforms.md (hunk 4 above)
Comment (brianhlin) [reply docs/release/supported_platforms.md]: I'll add 3.6 in another PR

**Pull request merged**